quaddicted-random-map.py

```diff
@@ -223,9 +227,9 @@ def _update_cache(self, key: str, value: str) -> None:
             json.dump(self.cache, cache_file_handle, indent=None)
 
     def _lowercase_files(self) -> None:
-        for filename in os.listdir(self.config.QUAKE_MAPS_PATH):
-            file_with_path = os.path.join(self.config.QUAKE_MAPS_PATH, filename)
-            os.rename(file_with_path, file_with_path.lower())
+        for filename in os.listdir(self.config.maps_path):
+            file_with_path = os.path.join(self.config.maps_path, filename)
+            # os.rename(file_with_path, file_with_path.lower())
```

Comment:
@Kartones this line is commented because it gives me the following error
`FileNotFoundError: [Errno 2] No such file or directory: '/home/sc/Repos/personal/quaddicted-random-map/vkquake/id1/maps/warpgate.bsp' -> '/home/sc/repos/personal/quaddicted-random-map/vkquake/id1/maps/warpgate.bsp'` 
but the bloody file does exists!!

Reply:
```
ls /home/sc/Repos/personal/quaddicted-random-map/vkquake/id1/maps/
AZSPQ1.BSP     coagula3_bone.txt  evildead.bsp     out1.bsp       sm18_blitz.bsp   sm27_bear.bsp     sm57_pulsar_se.bsp  sm7_inane.bsp    stores.txt
AZSPQ1.TXT     coagula3.bsp       evildead.txt     out1.txt       sm18_miikka.bsp  sm27_inertia.bsp  sm7_aard.bsp        sm7_pack.txt     TDA.bsp
AZSPQ2.BSP     DARK04B.TXT        gmsp3.txt        qmine.bsp      sm18_misyu.bsp   sm27_pack.txt     sm7_aru.bsp         sm7_rap.bsp      TDA.txt
celscape.bsp   DARKBASE.BSP       gmsp3v2.bsp      Qmine.txt      sm18_nane.bsp    sm27_ray.bsp      sm7_bal.bsp         sm7_ray.bsp      warpgate.bsp
celscape.txt   DARKHALL.BSP       ne_deadcity.bsp  shoggoth.bsp   sm18_pack.txt    sm27_than.bsp     sm7_bear.bsp        sm7_scampie.bsp  warpgate.txt
celscape.unix  dknight            ne_deadcity.txt  sm18_bear.bsp  sm27_ap.bsp      sm27_xenon.bsp    sm7_frib.bsp        stores.bsp
```

Reply:
still the whole thing works so far without lower casing the filenames ... 🤷‍♂️

Reply:
>  '/home/sc/Repos/personal/quaddicted-random-map/vkquake/id1/maps/warpgate.bsp' -> '/home/sc/repos/personal/quaddicted-random-map/vkquake/id1/maps/warpgate.bsp'

hehe, check your path: `/home/sc/Repos/` -> `/home/sc/repos/`  it lowercases the full path!

so let me fix that part on a different pr

Reply:
should be fixed now, if you pull latest master

Reply:
I am an idiot XD or at least blind 😎

Reply:
yup, it does work now!